core/kernel/persistence/smoothsql/class.Class.php

```diff
@@ -471,7 +472,7 @@ public function deleteInstances( core_kernel_classes_Class $resource, $resources
         		// as deleted.
         		$this->getPersistence()->exec($query);	
         		$returnValue = true;
-            } catch (PDOException $e) {
+            } catch (DBALException $e) {
```

Comment:
While PDO driver class still exists in Tao catching only `DBALException` may break some functionality. I would suggest two options:
1. Catch both of them
2. Log a warning message (which says that deprecated driver is configured) for any initialization of PDO driver in order to warn that it may lead to some inconsistent behavior.

Reply:
no longer exists in TAO